### ISSUE TITLE
Additional  information to learn

### ISSUE THREAD

**Issue by b03505036** [ultralytics/yolov5#5298]
Hi there,

I got a dataset with object detection label plus a global location label viewing from the top.

I wanna adding the global location label to the yolo v5 head and make it to learn well.

Any suggestion for this? (where the code should I modify or which the loss of  global location should I use)

It will be a big help.

Thank you a lot!

**Comment by glenn-jocher** [comment]
@b03505036 you'd have to modify the dataloader, loss function, and architecture to add additional regression values to each anchor. There's no tutorial for this but if you simply follow each regression, i.e. xywh, and expand it to additional lengths, i.e. 4 > 5 then that should work.
